lib/app-components/addon/components/project-contributors/list/item/template.hbs

```diff
@@ -107,9 +108,10 @@
     <div class='hidden-xs col-sm-2 text-center'>
         <button
             data-test-project-contributors-list-item-remove-button
-            {{action @removeContributor @contributor}}
+            type='button'
             class='btn btn-danger btn-sm'
             disabled={{not this.canRemove}}
+            {{action @removeContributor @contributor}}
```

Comment:
Can we use element modifiers here?

Reply:
at risk of creeping scope, maybe should consider enabling [`no-action`](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-action.md)?

Reply:
I think it's a good idea because of how confusing it's gotten. But as you say, let's defer enabling it to next iteration.